pkg/core/block/block.go

```diff
@@ -211,3 +211,30 @@ func (b *Block) UnmarshalJSON(data []byte) error {
 	}
 	return nil
 }
+
+// GetExpectedBlockSize returns expected block size which should be equal to io.GetVarSize(b)
+func (b *Block) GetExpectedBlockSize() int {
+	var transactionsSize int
+	for _, tx := range b.Transactions {
+		transactionsSize += tx.Size()
+	}
+	return b.GetExpectedBlockSizeWithoutTransactions(len(b.Transactions)) + transactionsSize
+}
+
+// GetExpectedBlockSizeWithoutTransactions returns expected block size without transactions size.
+func (b *Block) GetExpectedBlockSizeWithoutTransactions(txCount int) int {
+	size := 4 + // Version
+		util.Uint256Size + // PrevHash
+		util.Uint256Size + // MerkleRoot
+		8 + // TimeStamp
+		4 + // Index
+		1 + // PrimaryIndex
+		util.Uint160Size + // NextConsensus
```

Comment:
It basically is a `sizeof(header)`, we can calculate it once in some init function via `io.GetVarSize` instead of duplicating this size data here.

Reply:
Done, but there's a problem with block's witness. Its size depends on the validators count. I solved this with the new `blockchain.defaultBlockWitness` field, but not sure that it's a good decision.